lib/tracing-limit/src/lib.rs

```diff
@@ -381,22 +387,16 @@ impl Visit for RateLimitedSpanKeys {
 
 #[derive(Default)]
 struct LimitVisitor {
-    pub limit: Option<u64>,
+    pub limit: Option<bool>,
 }
 
 impl Visit for LimitVisitor {
-    fn record_u64(&mut self, field: &Field, value: u64) {
-        if self.limit.is_none() && field.name() == RATE_LIMIT_SECS_FIELD {
+    fn record_bool(&mut self, field: &Field, value: bool) {
+        if self.limit.is_none() && field.name() == RATE_LIMIT_FIELD {
```

Comment:
Do we need the `.is_none` guard? Is there a problem with just setting it twice (other than that's what we used to silently forbid)?

Reply:
I don't think there will be a problem with setting it twice since we default to false if the limit doesn't exist during the on_event method execution.